pubspec.yaml

```diff
@@ -4,7 +4,9 @@ description: A Flutter demo using the Twitch API.
 dependencies:
   flutter:
     sdk: flutter
-  twitch: ^0.3.0
+  twitch:
+    path: ../../lab/twitch.dart
```

Comment:
Needs to be updated to a proper dependency once we update the twitch client with the ability to fetch top vids